test/unit/Services/deposit/DepositTransactionService.spec.ts

```diff
@@ -0,0 +1,44 @@
+import {generateDeposit, DepositTx} from "../../../../src/renderer/services/deposit/DepositTransactionService";
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+import {Contract, ethers} from "ethers";
+import {deployDepositContract, generateKeyPair} from "./utils";
+// eslint-disable-next-line @typescript-eslint/no-require-imports,@typescript-eslint/no-var-requires
+const ganache = require("ganache-cli");
+
+describe("Deposit transaction service unit tests", () => {
+    const prKey = generateKeyPair(124);
+    const prKey2 = generateKeyPair(125);
```

Comment:
those are eth2 keys, you need eth1 keys to deploy contract and execute transaction